test/OrchardCore.Tests/ResourceManagement/ResourceManagerTests.cs

```diff
@@ -798,13 +798,13 @@ public async Task RenderLocalStyle()
         }
 
         #region Helpers
-        private async Task<IDocument> ParseHtmlAsync(IHtmlContent content)
+        private Task<IDocument> ParseHtmlAsync(IHtmlContent content)
         {
             using (var writer = new StringWriter())
             {
                 content.WriteTo(writer, HtmlEncoder.Default);
 
-                return await browsingContext.OpenAsync(res => res.Content(writer.ToString()).Address(basePath));
+                return browsingContext.OpenAsync(res => res.Content(writer.ToString()).Address(basePath));
```

Comment:
this escapes the using and writer can be disposed before await is done

Reply:
ah but the writer is not needed after this, so is fine 👍🏻

Reply:
or depends when the callback is called, so can be a bomb,  confusing 🤯

Reply:
Dont' remove it. There are more risks than benefits.

Reply:
@lahma I think you're right

Just didn't see the using, i'll revert this one, thanks